pkg/diskmaker/controllers/lvset/reconcile.go

```diff
@@ -165,6 +178,26 @@ DeviceLoop:
 				continue DeviceLoop
 			}
 		}
+
+		getTime := *r.deviceAgeMap.getTime
+
+		// skip devices younger than deviceMinAge, but remember them since they pass filters
+		if getTime().Sub(firstObserved) < deviceMinAge {
```

Comment:
if we rename function above then, this could be written as  - `if currentTime.After(availableAt) {}` .

Reply:
how about this? also removes `getTime`:
```
// get/set firstObserved
		isOldEnough := r.deviceAgeMap.isOlderThan(blockDevice.KName, deviceMinAge)

// 		// skip devices younger than deviceMinAge, but remember them since they pass filters
		if isOldEnough {
```